components/measurement/SummaryText.js

```diff
@@ -16,7 +16,7 @@ const SummaryText = ({
 }) => {
   const intl = useIntl()
   const metadata = getTestMetadata(testName)
-  const formattedDate = moment(date).format('LL')
+  // const formattedDate = moment(date).format('LL')
   const formattedDateTime = intl.formatDate(moment.utc(date).toDate(), {
```

Comment:
Can you try and do this without having to use the `moment.utc` call? In general we would like to move away from [moment](https://momentjs.com), so we are trying to refactor the code that uses the package. You can find more information about using [alternatives here](https://momentjs.com/docs/#/-project-status/).

If this isn't a straight forward change, don't worry about it. Just reply here that it is better to defer it to a future PR or something specific you have to say about it.

Reply:
Hey, thanks for reviewing! 
I have done the changes. I am looking into removing momentJS dependency altogether with something lightweight. Will it be okay if I open a future PR after removing this dependency from the project completely?

Also, I noticed I have committed changes directly to the master branch in this PR. I have made sure to make branches for the other PRs.